shared/episode.py

```diff
@@ -29,13 +25,13 @@ def next_frame(self):
     def load(self):
         pass
 
-    def save(self):
-        with open(last_episode_file, 'w') as f:
+    def save(self, path):
+        with open(path, 'w') as f:
             f.writelines([json.dumps(frame, default=fallback_to_int) for frame in self.frames])
```

Comment:
In tegenstelling tot wat je zou denken zet writelines geen newlines tussen de regels. 
En dat wil je wel voor een jsonl formaat. 

Moet dus zijn:
 f.write('\n'.join([json.dumps(frame, default=fallback_to_int) for frame in self.frames]))

Reply:
Ok. Is dat niet jouw code? Overigens denk ik dat je hier nog steeds een newline mist omdat je technisch op een newline wil eindigen.

Reply:
```The last character in the file may be a line separator, and it will be treated the same as if there was no line separator present.```

Dus blijkbaar niet. Je unix based editors gaan er waarschijnlijk nog steeds over zeuren dus misschien toch goed om te doen.

Reply:
Dan zou het dit moeten zijn:
f.writelines[json.dumps(frame, default=fallback_to_int)+'\n' for frame in self.frames])